app/models/keyword.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Keyword
+  attr_reader :id, :name
+
+  def initialize(id, name)
+    @id = id
+    @name = name
+  end
```

Comment:
```suggestion
  attr_accessor :id, :name
```

Does it work with just `attr_accessor`? (i.e. getter and setter)

Reply:
It works with `attr_accessor` but lint catches this because we are not setting the values, so `attr_reader` to limit the write access to these values sound better for me.

Reply:
Lint is here to help us pay attention to the right things. But it is not here to lower the quality of the code or to make Fabricator more complex than what they should be 😇 

`keywords` aims to be an ActiveRecord model in the future. To have the least need for changes, `attr_accessor` will be used by the active record and therefore the way all other libraries are optimized (e.g. Fabrication).

While it's `working` with attr_reader and an initializer, it makes tests less standard and it will require you to refactor the code more later. That's the reason why I recommend using `attr_accessor` in this case.

Reply:
I see, thank you for the insight, I will update this model to `attr_accessor` to be future proof.